code/modules/mob/mob.dm

```diff
@@ -1697,6 +1709,30 @@ Use this proc preferably at the end of an equipment loadout
 			src.callOnFace -= .
 
 
+/mob/proc/StartMoving()
+	var/datum/listener
+	for(. in src.callOnStartMove)
```

Comment:
This is a really confusing usage of `.` for very little gain... Use a normal variable here.